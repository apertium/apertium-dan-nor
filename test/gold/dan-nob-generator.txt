[1XR8ugqC8dyr]
Hun-vandrefalken, som er størst, klarer å ta bytte på helt opp til 1 kg. [/option]
[/1XR8ugqC8dyr]
[yCVw30jwzGCn]
Den er en av de foretrukne fuglene blant falkonerer. [/option]
[/yCVw30jwzGCn]
