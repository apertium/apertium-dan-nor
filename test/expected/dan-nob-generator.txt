[-Gmsv6GmJNSB#0] Det er med til å forklare, hvorfor denne politikken får så stor en viktighet innenfor EU.
[/-Gmsv6GmJNSB]
[06Jrtnx4CU6Z#0] Efterhvert som der er kommet mer styr på bruken av miljøgifter generelt, men især i landbruket, sees vandrefalker også i Danmark i et stadig større antall på trekk mot Norge og Sverige.
[/06Jrtnx4CU6Z]
[0ebRCh62wmL_#0] De ble vurdert tvert imot utelukkende i deres egenskap som isolerte grunnstykker.
[/0ebRCh62wmL_]
[13lkKfMMjli-#0] Vandrefalkens føde består av mellomstore fugler som den fanger i luften.
[/13lkKfMMjli-]
[1IulJoMbI3rr#0] Her hevdes den å kunne nå en fart på 390 km\/t.
[/1IulJoMbI3rr]
[1JkG8hoTNN4r#0] Langt det meste av all menneskelig aktivitet foregikk, før vi lærte at skrive.
[/1JkG8hoTNN4r]
[1XR8ugqC8dyr#0] Hun-vandrefalken som er størst, klarer at ta bytte på helt opp til 1 kg.
[/1XR8ugqC8dyr]
[1w4SjZ34bVt1#0] Vandrefalken var gjennom 1960-årene og 1970-årene sterkt i tilbakegang i Skandinavia og mange andre steder.
[/1w4SjZ34bVt1]
[2iUkU1dA7rtZ#0] Det dreier seg ofte om duer, vadefugler, mindre ender, kråkefugler og troster.
[/2iUkU1dA7rtZ]
[5MNSXDeaOmHl#0] Gjengen slimsvamper er tradisjonelt blitt plassert i svampgjengen, men de adskiller seg fra andre svamptyper på vesentlige punkt; de mangler slik cellevegger og kan på samme måte som amøber oppta fast føde ved *fagocytose.
[/5MNSXDeaOmHl]
[5x8TEJ_RVxFV#0] Den 30. juni ble tre unger (to hanner og en hun) ringmerket nær Hammerhus på Bornholm.
[/5x8TEJ_RVxFV]
[6jYihw37Zx5J#0] mer verdifull
[/6jYihw37Zx5J]
[7Iu3xZgFBVRz#0] Vandrefalken gjorde sitt siste yngleforsøk i Danmark i 1972, men er nå tilbake som dansk ynglefugl, ettersom den siden 2002 har ynglet på Møns *klint (2002: 2 unger, 2003: 3 unger).
[/7Iu3xZgFBVRz]
[8jRQMBgLNpMl#0] mest borgerlige
[/8jRQMBgLNpMl]
[94n3qPHdw-OR#0] Universell anestesi medfører muskellapphet og hemmer pusten, derfor må denne i de fleste tilfeller understøttes eller overtas i den tiden.
[/94n3qPHdw-OR]
[9qPDzahWsJRS#0] Vandrefalk fra England.
[/9qPDzahWsJRS]
[AGf_CIcG6bLB#0] Den er kjent for at være blant verdenens hurtigste fugler, når den styrtdykkeren mot et bytte.
[/AGf_CIcG6bLB]
[CGIBBF-qJiQP#0] Vandrefalk (*Falco *peregrinus) er en rovfugl som er utbredt over alle kontinent på nær Antarktis.
[/CGIBBF-qJiQP]
[CnnBLgiFYFHu#0] Fra 1985 ble påbegynt i Danmark undersøkelse av alt donorblod, derfor er denne smitteveien nå opphørt.
[/CnnBLgiFYFHu]
[Ew_IY77ZjK13#0] De ble vurdert
[/Ew_IY77ZjK13]
[GuCJGMSC55jy#0] I 2009 ble der for første gang siden 1937 (hvor tre unger ble ringmerket på Bornholm) ringmerket *vandrefalkeunger i Danmark.
[/GuCJGMSC55jy]
[ILH_TKWtSLEy#0] Den 18. mai ble der ringmerket tre unger på Stevns Klint (to hanner og en hun) og den 19. mai fire unger på Møns Klint (to hanner og to huner).
[/ILH_TKWtSLEy]
[Iy3D9K4sfVe8#0] Der var senest i 2012 12 ynglepar i Danmark, hvorav/hvor av 6 par hadde suksess med å få unger på vingene.
[/Iy3D9K4sfVe8]
[JII-WAQXpCWP#0] Betegnelsen katolsk ble forstått i motsetning til *særmeninger og kjetteri, altså i den *nikæno-konstantinopolitanske konfesjon.
[/JII-WAQXpCWP]
[JXsiSKP2HO9_#0] I 2004 forsøkte en nytt dansk *vandrefalkepar å yngle på Bornholm, men det mislyktes dessverre pga. manglende beskyttelse av yngleområdet.
[/JXsiSKP2HO9_]
[MvI9A0AN1aXS#0] Vandrefalken har siden 2007 ynglet med suksess på Stevns Klint (2007: 2 unger, 2008: 3 unger) og den har også etablert rede i 2009.
[/MvI9A0AN1aXS]
[UDEmh40X_Na9#0] av
[/UDEmh40X_Na9]
[WGUkSKGKhcJ3#0] Især insekticidet DDT SOM medførte *tyndskallede egg, påvirkede vandrefalkens reproduksjonsevne.
[/WGUkSKGKhcJ3]
[WrXo3zkuSdPE#0] *Inskriften på *Jellingstenen lyder slik:
[/WrXo3zkuSdPE]
[X2KfDHUw343q#0] mest borgerlig
[/X2KfDHUw343q]
[XIV_kHEp8XGA#0] Ynglebestanden er vurdert som sårbar på den danske rødlisten.
[/XIV_kHEp8XGA]
[XZnUbh3Z2JLN#0] Miljøgifter var den store synderen da de ødela fuglenes fruktbarhet.
[/XZnUbh3Z2JLN]
[XrLP3qOkI-cd#0] Ynglefugl i Danmark.
[/XrLP3qOkI-cd]
[YB3NJdSjwuh8#0] Miljøgifter.
[/YB3NJdSjwuh8]
[YiITmoSuWXBn#0] Dette aerodynamiske designet er også blitt anvendt av flykonstruktører for å øke kampflys hastighet.
[/YiITmoSuWXBn]
[e_tufqM_NiFZ#0] Det finnes omkring 16 underarter.
[/e_tufqM_NiFZ]
[gJh2RMTY1F_N#0] Wikipedia er en internasjonal internettbasert encyklopedi som utgis av den ideelle organisasjonen Wikimedia Foundation
[/gJh2RMTY1F_N]
[gchkSnxi0muM#0] Fuglen danner med sine *smalle vinger en pilformet profil når den flyr.
[/gchkSnxi0muM]
[gqY0gQVo4cAd#0] Katolske kirker er de kristne kirkene som anerkjenner pavens primat.
[/gqY0gQVo4cAd]
[nxBjdo-MxJ2a#0] Viruset overføres aldri ved sosial kontakt, berøring eller kyss, likesom det heller ikke er holdepunkt for overførsel via insektstikk, mat eller drikker.
[/nxBjdo-MxJ2a]
[ro_zQTI_DdSW#0] Zoologen Rachel Carson påpekte nettopp dette problemet allerede i 1962 i sin bok *Silent Spring.
[/ro_zQTI_DdSW]
[s4bPT-5sqDDz#0] Hvis det ikke før fødselen innstiller seg på langs, vil det ikke kunne fødes, derfor krever denne situasjonen uvegerlig forløsning ved keisersnitt.
[/s4bPT-5sqDDz]
[yCVw30jwzGCn#0] Den er en av de foretrukne fuglene blant falkonerer.
[/yCVw30jwzGCn]
